### PR TITLE
Add synonym support for slim location description

### PR DESCRIPTION
**Pull request by oleg-n-k**
Add synonym support for slim location description

### CODE REVIEW

**Review by AlexeyShok — APPROVED** [review]

File: HappyTravel.Nakijin.Api/Services/LocationMappingInfo/SlimLocationDescriptionService.cs [hunk 1]
```diff
@@ -33,9 +34,11 @@ public async Task<Result<SlimLocationDescription>> GetForCountry(int countryId,
 
         public async Task<Result<SlimLocationDescription>> GetForLocality(int localityId, string languageCode)
         {
-            var locality = await _context.Localities
-                .FirstOrDefaultAsync(x => x.Id == localityId && x.IsActive);
-                
+            var locality = await _context.Localities.Include(l => l.LocalitySynonyms)
+                .Where(l => l.Id == localityId || l.LocalitySynonyms.Any(s => s.ExLocalityId == localityId))
```
Comment (hasmikharutyunyan) [inline HappyTravel.Nakijin.Api/Services/LocationMappingInfo/SlimLocationDescriptionService.cs:38]: why needed 2 `Where` - s ?

File: HappyTravel.Nakijin.Api/Services/LocationMappingInfo/SlimLocationDescriptionService.cs [hunk 2]
```diff
@@ -33,9 +34,11 @@ public async Task<Result<SlimLocationDescription>> GetForCountry(int countryId,
 
         public async Task<Result<SlimLocationDescription>> GetForLocality(int localityId, string languageCode)
         {
-            var locality = await _context.Localities
-                .FirstOrDefaultAsync(x => x.Id == localityId && x.IsActive);
-                
+            var locality = await _context.Localities.Include(l => l.LocalitySynonyms)
```
Comment (oleg-n-k) [inline HappyTravel.Nakijin.Api/Services/LocationMappingInfo/SlimLocationDescriptionService.cs:37]: for code readability

File: HappyTravel.Nakijin.Api/Services/LocationMappingInfo/SlimLocationDescriptionService.cs (hunk 1 above)
Comment (oleg-n-k) [reply HappyTravel.Nakijin.Api/Services/LocationMappingInfo/SlimLocationDescriptionService.cs:38]: for code readability

Comment (hasmikharutyunyan) [reply HappyTravel.Nakijin.Api/Services/LocationMappingInfo/SlimLocationDescriptionService.cs:38]: I think if will separate each condition on a new line will be readable, but it is not a problem for me

**Review by hasmikharutyunyan — APPROVED** [review]

File: HappyTravel.Nakijin.Api/Services/LocationMappingInfo/SlimLocationDescriptionService.cs (hunk 1 above)
Comment (oleg-n-k) [reply HappyTravel.Nakijin.Api/Services/LocationMappingInfo/SlimLocationDescriptionService.cs:38]: ok

**Pull request merged**